python/ray/tune/utils/util.py

```diff
@@ -284,7 +284,9 @@ def deep_update(original,
 
 
 def flatten_dict(dt, delimiter="/", prevent_delimiter=False):
-    dt = copy.deepcopy(dt)
+    """Flatten dict. Output and input are of the same dict type.
+    Input dict remains the same after the operation."""
```

Comment:
Let's use the Google python docstrings?

Specifically, we need to keep 1 sentence (no overflow) in the top line:
```suggestion
    """Flatten dict. 
    
    Output and input are of the same dict type.
    Input dict remains the same after the operation.
    """
```

Reply:
Got it :)